### ISSUE TITLE
"No Policy" Context access permission bug 

### ISSUE THREAD

**Issue by neilness** [modxbot/migrate#10398]
neilness created Redmine issue ID 10398

If you edit an existing Context access permission, and select "No Policy", you aren't prompted that it is an invalid selection, and it saves to the DB as policy => 0. You will then get a "loading" message indefinitely, and policies won't be displayed or accessible in the manager until you change the value in the database manually.

**Comment by splittingred** [comment]
splittingred submitted:

I can't replicate this at all - mine shows an error message saying "No policy specified.". Can you detail the steps you took to do this?

**Comment by neilness** [comment]
neilness submitted:

Strange! I first noticed this issue in a fresh install of 2.0.0-pl, and i'm still having the issue after upgrading to 2.0.3-pl. I've tested on the latest versions of Mac Chrome, Firefox, & Safari, and Windows Chrome, Firefox and IE. Sure thing: 1) Edit Context --> Access Permissions --> Edit (Or create new) individual entry
2) Change Access Policy to "(no policy)"
(note: If you do NOT changing the blank "Access Policy" Field, it will not allow you to save, only if you specifically set it to "(no policy)".
3) Click Save --> "Saving" Popup --> Grid tries to refresh, says "Loading" indefinitely
4) After refreshing the page, no entries show up in the grid, and you cannot add new entries.
5) If you delete the entry in the database, or change it to a value > 0, the grid will display again in the manager. If you'd like to try on my install, I've created test accounts for the manager and phpmyadmin, and i can email the log in details. ALSO - Important! I noticed that if you follow the above steps in a context other than "web", the context disappears from the Resource Tree, and you get an "Access Denied" Error.
